plugins/email_service/spec/controllers/configsets_controller_spec.rb

```diff
@@ -15,38 +15,37 @@
       'Project', default_params[:domain_id], default_params[:project_id],
       default_params[:project_id]
     )
+    puts "\n ==============================================================\n"
```

Comment:
I am still seeing the puts. This not ok. It will log in production.

Reply:
removed.